src/main/resources/project/jython/stop_cluster.jython

```diff
@@ -38,15 +39,46 @@ clusterName = r'''
 [% cluster_name %]
 '''.strip()
 
+timeout = r'''
+[% timeout %]
```

Comment:
1) .jython templates are designed for internal use.
2) shutdownTimeout parameter is being modified at the perl side of procedure, it is not a regular property forwarding, as forceShutdown or ignoreSessions.